### PR TITLE
Xpetra/MueLu: Commits related to #2630

### PR DESCRIPTION
**Pull request by csiefer2**
Auto-PR for SHA f3992fa

### CODE REVIEW

File: packages/xpetra/src/CrsMatrix/Xpetra_EpetraCrsMatrix.hpp [hunk 1]
```diff
@@ -1015,9 +1006,6 @@ class EpetraCrsMatrixT <int, EpetraNode>
             // End of TMP
 
             out << "Node number of entries = " << getNodeNumEntries() << std::endl;
-            if (isFillComplete()) {
-              out << "Node number of diagonals = " << getNodeNumDiags() << std::endl;
-            }
             out << "Node max number of entries = " << getNodeMaxNumRowEntries() << std::endl;
```
Comment (mhoemmen) [inline packages/xpetra/src/CrsMatrix/Xpetra_EpetraCrsMatrix.hpp]: This reminds me that the ability to call getNodeMaxNumRowEntries or getNodeNumEntries before first fillComplete of a CrsGraph is one of the few things that makes it not thread safe to add entries to a graph.

Comment (mhoemmen) [reply packages/xpetra/src/CrsMatrix/Xpetra_EpetraCrsMatrix.hpp]: That's not relevant to this PR but it's good to keep in mind :D .

File: packages/xpetra/src/CrsMatrix/Xpetra_EpetraCrsMatrix.hpp [hunk 2]
```diff
@@ -1992,9 +1973,6 @@ class EpetraCrsMatrixT <long long, EpetraNode>
...
```
Comment (mhoemmen) [inline packages/xpetra/src/CrsMatrix/Xpetra_EpetraCrsMatrix.hpp]: See above note (not relevant to this PR though).

**Comment by mhoemmen** [comment]
@csiefer2 How come I can't merge this?  The "Pre-Merge Inspection" says that it's "NOT INSPECTED YET - WARNING: NO REVIEWS" but I just reviewed and approved it.

**Comment by allevin** [comment]
@mhoemmen After a review is performed, it takes a little bit of time for the autotester to come back around and see the changes and adjust the status flags.

**Pull request merged**